src/Operation/Append.php

```diff
@@ -14,28 +14,27 @@
 use loophp\collection\Iterator\MultipleIterableIterator;
 
 /**
- * @psalm-template TKey
- * @psalm-template TKey of array-key
- * @psalm-template T
+ * @template TKey
+ * @template T
  */
 final class Append extends AbstractOperation
 {
     /**
-     * @psalm-return Closure(T...): Closure(Iterator<TKey, T>): Iterator<int|TKey, T>
+     * @return Closure(mixed ...$items): Closure(Iterator<TKey, T>): Iterator<int|TKey, T|mixed>
```

Comment:
the changes here bring it in line with the interface since items of different types can be appended

Reply:
Are you sure about that? Shouldn't it be: 

```suggestion
     * @return Closure(...mixed): Closure(Iterator<TKey, T>): Iterator<int|TKey, T|mixed>
```

instead ?

Reply:
Both are valid, I made a similar change here in previous [PR](https://github.com/loophp/collection/blob/master/src/Collection.php#L151). Also examples from the issue I submitted to PHPStan 😄 :
https://phpstan.org/r/474314ca-f343-4454-b100-dda6fb144bd2
https://psalm.dev/r/12f59e9b5e

The benefit of using the name of the parameter is that it makes it easier when reading to tie it to the $items [here](https://github.com/loophp/collection/blob/master/src/Operation/Append.php#L30)

Reply:
Actually, the other version (`...mixed`) is not valid for PHPStan, only for Psalm. Whereas looks like this one is valid for both 🤷

Reply:
Ho man, nice discovery! Going to use it everywhere !